oauth2/oauth2_auth_code_test.go

```diff
@@ -1192,6 +1204,79 @@ func TestAuthCodeWithDefaultStrategy(t *testing.T) {
 				t.Run("strategy=jwt", run("jwt"))
 			})
 
+			t.Run("case=can revoke token chains with ID obtained from consent requests", func(t *testing.T) {
```

Comment:
It would be good to test a real "chain", i.e. `len(chain) > 1` here. Especially since we have the graceful refresh, we can get forks in the chain and I'd expect to have all branches revoked as well, right?

Reply:
https://github.com/ory/hydra/issues/3941